src/detectors/BTOF/BTOF.cc

```diff
@@ -52,13 +52,14 @@ void InitPlugin(JApplication* app) {
       },
       app)); // Hit reco default config for factories
 
-  app->Add(new JOmniFactoryGeneratorT<LGADChargeSharing_factory>(
-      "LGADChargeSharing", {"TOFBarrelHits"}, {"TOFBarrelSharedHits"},
-      {.sigma_sharingx        = 0.1 * dd4hep::cm,
-       .sigma_sharingy        = 0.5 * dd4hep::cm,
-       .readout               = "TOFBarrelHits",
-       .same_sensor_condition = "sensor_1 == sensor_2",
-       .neighbor_fields       = {"x", "y"}},
+  app->Add(new JOmniFactoryGeneratorT<SiliconChargeSharing_factory>(
+      "TOFBarrelChargeSharing", {"TOFBarrelHits"}, {"TOFBarrelSharedHits"},
```

Comment:
Many factories (for historical reasons) use the primary output collection as the name. Consistency helps when changing factory properties.
```suggestion
      "TOFBarrelSharedHits", {"TOFBarrelHits"}, {"TOFBarrelSharedHits"},
```